bdai_ros2_wrappers/bdai_ros2_wrappers/static_transform_broadcaster.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2023 Boston Dynamics AI Institute Inc.  All rights reserved.
+from typing import Dict, Iterable, Optional, Union, cast
+
+from geometry_msgs.msg import TransformStamped
+from rclpy.node import Node
+from rclpy.qos import DurabilityPolicy, HistoryPolicy, QoSProfile
+from tf2_msgs.msg import TFMessage
+
+
+class StaticTransformBroadcaster:
+    """
+    A modified :class:`tf2_ros.StaticTransformBroadcaster` that stores transforms
+    sent through it, matching ``rclcpp::StaticTransformBroadcaster`` behavior.
+    """
+
+    DEFAULT_QOS = QoSProfile(depth=1, durability=DurabilityPolicy.TRANSIENT_LOCAL, history=HistoryPolicy.KEEP_LAST)
+
+    def __init__(self, node: Node, qos: Optional[Union[QoSProfile, int]] = None) -> None:
+        """
+        Constructor.
+
+        :param node: The ROS2 node.
+        :param qos: A QoSProfile or a history depth to apply to the publisher.
+        """
+        if qos is None:
```

Comment:
what if `qos == 0`?  do we want that to be the same as the `None` case?

Reply:
I subjectively believe it shouldn't be treated as such. 0 means no queue (and may result in an error, though I'm not sure). None means no qos provided, use defaults.